charts/astronomer/templates/houston/api/houston-deployment.yaml

```diff
@@ -111,6 +114,11 @@ spec:
             initialDelaySeconds: {{ .Values.houston.readinessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.houston.readinessProbe.periodSeconds }}
             failureThreshold: {{ .Values.houston.readinessProbe.failureThreshold }}
+          args:
```

Comment:
let's add feature flag here. I'm 100% sure all clients want it.

Reply:
looks good, please add tests to you PR.

Reply:
Agreed about tests. Looks good, but tests will make it much easier to grok.